api/v1/views/index.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python3
+""" ssss """
+from api.v1.views import app_views
+from flask import jsonify
+from models import storage
+
+
+@app_views.route('/status', strict_slashes=False)
+def status():
+    """ sss """
```

Comment:
Maybe sss is not a good description for what this function does

Reply:
Done!